fattmerchant_ios_sdkTests/Usecase/TakePaymentTests.swift

```diff
@@ -32,11 +32,13 @@ class TakePaymentTests: XCTestCase {
 
   func testCreateChargeRequest() {
     let amount = Amount(dollars: 10.00)
+    let poNumber = "asdf4terstges"
```

Comment:
I think we also need to add another test case which includes empty/null poNumber to check how the behaviour would go.

Reply:
> Please also add screenshots or a video showing what has been done in this PR/Ticket, specially for bug fixes, for easier review and to be as a proof of your fix that it worked as expected.

I don't have card reader to be able to get picture of the transaction.. rather I can test with ensuring the test case work fine...